### PR TITLE
Homework-React -week3

### PR DESCRIPTION
**Pull request by mahieakhtar**
Homework-React -week3

### CODE REVIEW

File: react/week3/src/Todo.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+import React from "react";
+
+const Todo = (props) => {
+ 
+  return (
+    
+    <form id="myForm" onSubmit ={props.onSubmit} style={{ paddingLeft: 40, marginTop: 16 }}>
```
Comment (Daryan-Tech) [inline react/week3/src/Todo.js]: Please try not to use inline styles in your react component

File: react/week3/src/TodoForm.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React, {useState} from 'react';
+
+const TodoForm = (props) => {
+  
+  const [checked, setChecked] = useState(false)
+  const handleClick = () => setChecked(!checked)
+
+  return (
+    <li style = {{ textDecoration: checked ? "line-through" : "" , border : "1px solid black", padding:"5px", width:"500px"}}>
```
Comment (Daryan-Tech) [inline react/week3/src/TodoForm.js]: alos here don't use inline styles,
so you will need 2 classes for this code 
you can install classnames from npm => https://jedwatson.github.io/classnames/
and do something like this instead
className={{ checked &&  "addNewClassName", "yourDefaultClassName" }}
"yourDefaultClassName" in you css file will do the default styles 
and "addNewClassName" you will add in and do the styles when checked is true

**Comment by Daryan-Tech** [comment]
good job please try to have a look at the second comment and see if you can do what I have suggested if you couldn't figure it out tell me so I can help you with it.

File: react/week3/src/Todo.js (hunk 1 above)
Comment (mahieakhtar) [reply react/week3/src/Todo.js]: Done.Thank you

File: react/week3/src/TodoForm.js (hunk 2 above)
Comment (mahieakhtar) [reply react/week3/src/TodoForm.js]: Done. Thank you

**Pull request merged**